language/stdlib/modules/libra_system.mvir

```diff
@@ -222,8 +238,17 @@ module LibraSystem {
         }
     }
 
-  // Get the address of the i'th validator.
-  public get_ith_validator_address(i: u64): address acquires ValidatorSet {
+    // Get the ValidatorInfo for the ith validator
+    public get_ith_validator_info(i: u64): Self.ValidatorInfo acquires ValidatorSet {
+      let vector_ref: &Vector.T<Self.ValidatorInfo>;
```

Comment:
white space seems inconsistent between 2 and 4 spaces

Reply:
Yeah, this is a problem... Hopefully we can add an autoformatter in the forthcoming Move source lang (send pleas to @tnowacki)!